nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/QueryRecord.java

```diff
@@ -102,6 +104,10 @@
 @DynamicProperty(name = "The name of the relationship to route data to", value="A SQL SELECT statement that is used to determine what data should be routed to this "
         + "relationship.", supportsExpressionLanguage=true, description="Each user-defined property specifies a SQL SELECT statement to run over the data, with the data "
         + "that is selected being routed to the relationship whose name is the property name")
+@WritesAttributes({
+    @WritesAttribute(attribute = "mime.type", description = "Sets the mime.type attribute to the MIME Type specified by the Record Writer"),
+    @WritesAttribute(attribute = "record.count", description = "The number of records selected by the query")
+})        
```

Comment:
Checkstyle failed with trailing whitespace here
`[WARNING] src/main/java/org/apache/nifi/processors/standard/QueryRecord.java[110] (regexp) RegexpSinglelineJava: Line has trailing whitespace.`

Otherwise LGTM, thanks! Will merge after contrib-check passes.